assets/queries/terraform/azure/storage_share_file_allows_all_acl_permissions/query.rego

```diff
@@ -0,0 +1,25 @@
+package Cx
+
+CxPolicy[result] {
+	resource := input.document[i].resource.azurerm_storage_share_file[name]
+
+	storage_share := resource.storage_share_id
+	attributeSplit := split(storage_share, ".")
+	allows_all_permissions(attributeSplit[1], input.document[i].resource)
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("azurerm_storage_share_file[%s].storage_share_id", [name]),
```

Comment:
maybe here is better to point to `azurerm_storage_share[%s].acl.access_policy.permissions` since it is here that changes need to be made.